framework/include/mfem/auxkernels/MFEMCrossProductAux.h

```diff
@@ -0,0 +1,53 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifdef MOOSE_MFEM_ENABLED
+#pragma once
+
+#include "MFEMAuxKernel.h"
+#include "mfem.hpp"
+
+/**
+ * Project s(x) * (U x V) onto a vector MFEM auxvariable.
+ *
+ * Parameters:
+ *   - variable (AuxVariableName, required): AuxVariable name (inherited)
+ *   - u (VariableName, required): vector MFEM variable U (vdim = 3)
+ *   - v (VariableName, required): vector MFEM variable V (vdim = 3)
+ *   - scale_factor (mfem::real_t, default=1.0): constant multiplier s(x)=scale_factor
+ *
+ * Notes:
+ *   - Requires variable to be L2_FECollection with map_type = INTEGRAL.
+ *   - Currently supports only interior DOFs (no shared/constrained DOFs).
```

Comment:
are these limitations of this kernel in particular or any of the MFEM auxkernels?